std_msgs/Header header
geometry_msgs/Point[16] poses
geometry_msgs/Point[4] vel